sale_channel/models/sale_channel.py

```diff
@@ -10,3 +10,8 @@ class SaleChannel(models.Model):
 
     name = fields.Char(required=True)
     active = fields.Boolean(default=True)
+    type_channel = fields.Selection(
+        [], help="Allows to use specific fields and actions for specific channel's type"
+    )
```

Comment:
I prefer 'channel_type' on my side, but I guess it is just a matter of taste :)

Reply:
it can be channel_type for sure! :)